cdap-app-fabric/src/main/java/io/cdap/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -235,8 +235,6 @@ protected void configure() {
                                servicesNamesBinder.addBinding().toInstance(Constants.Service.LOGSAVER);
                                servicesNamesBinder.addBinding().toInstance(Constants.Service.TRANSACTION_HTTP);
                                servicesNamesBinder.addBinding().toInstance(Constants.Service.RUNTIME);
-                               servicesNamesBinder.addBinding()
-                                 .toInstance(Constants.AppFabricHealthCheck.APP_FABRIC_HEALTH_CHECK_SERVICE);
```

Comment:
If the `Constants.AppFabricHealthCheck` class is no longer needed, remove it from `Constants.java` as well.